src/content/en/showcase/2016/service-worker-perf.md

```diff
@@ -0,0 +1,679 @@
+
+The service worker's status is not a dimension Google Analytics provides by default; however, Google Analytics does give you the ability to create your own [custom dimensions](https://support.google.com/analytics/answer/2709828) and define them however you want.
+
+For IOWA, we created a custom dimension called *Service Worker Status* and set its scope to [hit](https://support.google.com/analytics/answer/2709828#example-hit) (i.e. per-interaction).&nbsp;<sup><a href="#footnotes">[4]</a></sup> Each custom dimension you create in Google Analytics is given a unique index within that property, and in your tracking code you can reference that dimension by its index. For example, if the index of the dimension we just created were 1, we could update our logic as follows to send the `firstpaint` event to include the service worker status:
+
+<pre class="prettyprint">
+ga('send', 'event', {
+  eventCategory: 'Performance',
+  eventAction: 'firstpaint',
+  // Rounds to the nearest millisecond since
+  // event values in Google Analytics must be integers.
+  eventValue: Math.round(timeToFirstPaint)
+  // Sends this as a non-interaction event,
+  // so it doesn't affect bounce rate.
+  nonInteraction: true,
+
+  <strong>// Sets the current service worker status as the value of
+  // `dimension1` for this event.
+  dimension1: getServiceWorkerStatus()</strong>
+});
+</pre>
+
+This works, but it will only associate the service worker's status with this particular event. Since *Service Worker Status* is something that's potentially useful to know for any interaction, it's best to include it with all data sent to Google Analytics.
+
+To include this information in all hits (e.g. all pageviews, events, etc.) we [set](/analytics/devguides/collection/analyticsjs/accessing-trackers#updating_tracker_data) the custom dimension value on the [tracker](/analytics/devguides/collection/analyticsjs/creating-trackers) object itself, prior to sending any data to Google Analytics.
+
+<pre class="prettyprint">
+ga('set', 'dimension1', getServiceWorkerStatus());
+</pre>
+
+Once set, this value gets sent with all subsequent hits for the current pageload. If the user load the page again later, a new value will likely be returned from the `getServiceWorkerStatus` function, and that value will be set on the tracker object.
```

Comment:
When referring to a function or method in text, include the '()'. Please double-check throughout.